webapp/src/selectors/index.tsx

```diff
@@ -2,10 +2,16 @@ import plugin_constants from 'plugin_constants';
 
 const pluginPrefix = `plugins-${plugin_constants.pluginId}`;
 
+// TODO: create a type for global state
```

Comment:
It is already created called `GlobalState` and you can import it from `mattermost-redux`.

Reply:
yes this has been fixed here https://github.com/Brightscout/mattermost-plugin-azure-devops/pull/28